test/extended/dr/quorum_restore.go

```diff
@@ -229,20 +233,20 @@ var _ = g.Describe("[sig-etcd][Feature:DisasterRecovery][Disruptive]", func() {
 				}
 
 				framework.Logf("Force new revision of etcd-pod")
-				_, err = oc.AdminOperatorClient().OperatorV1().Etcds().Patch(context.Background(), "cluster", types.MergePatchType, []byte(`{"spec": {"forceRedeploymentReason"  :   "recover-etcd"}}`), metav1.PatchOptions{})
+				_, err = oc.AdminOperatorClient().OperatorV1().Etcds().Patch(context.Background(), "cluster", types.MergePatchType, []byte(`{"spec": {"forceRedeploymentReason": "recover-etcd"}}`), metav1.PatchOptions{})
 				o.Expect(err).NotTo(o.HaveOccurred())
 
 				framework.Logf("Force new revision of kube-apiserver")
-				_, err = oc.AdminOperatorClient().OperatorV1().KubeAPIServers().Patch(context.Background(), "cluster", types.MergePatchType, []byte(`{"spec": {"forceRedepl  oymentReason": "recover-kube-apiserver"}}`), metav1.PatchOptions{})
+				_, err = oc.AdminOperatorClient().OperatorV1().KubeAPIServers().Patch(context.Background(), "cluster", types.MergePatchType, []byte(`{"spec": {"forceRedeploymentReason": "recover-kube-apiserver"}}`), metav1.PatchOptions{})
 				o.Expect(err).NotTo(o.HaveOccurred())
 
 				framework.Logf("Wait for etcd pods to become available")
-				_, err = exutil.WaitForPods(
+				_, err = waitForPodsTolerateClientTimeout(
```

Comment:
wait, why do you have to tolerate timeouts here?  at this point etcd should be available?

Reply:
I am forcing a revision of both etcd and apiserver static pod resources. This causes disruption as we transfer leadership.  I will try to dial this in but it has proven a challenge. The error comes from apiserver.